shared/src/main/scala/coop/rchain/catscontrib/SyncInstances.scala

```diff
@@ -1,18 +1,15 @@
-package coop.rchain.node
+package coop.rchain.catscontrib
 
 import cats.data.EitherT
 import cats.effect.{ExitCase, Sync}
 import cats.{Applicative, FlatMap}
-import coop.rchain.comm.{CommError, UnknownCommError}
 import monix.eval.Task
 
-//TODO investigate if this can be extracted to shared, the problem is errors
-//ATM the project does not have a notion of shared base error which this could require
 object SyncInstances {
-  type CommErrT[F[_], A] = EitherT[F, CommError, A]
-  type Effect[A]         = CommErrT[Task, A]
 
-  def syncEffect: Sync[Effect] =
+  def syncEffect[E](toThrowable: E => Throwable,
+                    fromThrowable: Throwable => E): Sync[EitherT[Task, E, ?]] = {
```

Comment:
This is hacky :)

Reply:
Why on earth even have anything like that? I mean this whole `syncEffect` function?
There is already  a `Sync[Task]` correct? Create an instance for `Sync[EitherT[F, E, ?]` where there is `Sync[F]`

Reply:
@goral09 BIO BIO BIO...

Reply:
@rabbitonweb I had this chat with @goral09 a few days ago, it's because `Sync[Task]` is broken due to our mixed-matched cats dependencies and I gave an example: https://github.com/rchain/rchain/pull/1225#discussion_r206863631